_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1101,6 +1103,18 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
         MFX_ERR_DEVICE_FAILED);
 
     // Configuration
+    std::vector<VAConfigAttrib> attrib_priority(1);
+    attrib_priority[0].type = VAConfigAttribContextPriority;
+    vaSts = vaGetConfigAttributes(m_vaDisplay,
+                          vaParams.profile,
+                          vaParams.entrypoint,
+                          attrib_priority.data(), (mfxI32)attrib_priority.size());
+
+    MFX_CHECK(!(VA_STATUS_ERROR_UNSUPPORTED_ENTRYPOINT == vaSts ||
+                VA_STATUS_ERROR_UNSUPPORTED_PROFILE == vaSts),
+                MFX_ERR_UNSUPPORTED);
+    m_MaxContextPriority = attrib_priority[0].value;
```

Comment:
Generally speaking attribute might not be supported by the driver. You must check this via VA_ATTRIB_NOT_SUPPORTED. See other examples in the code... Like this one:
```
_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp:1023:    if (attrs[ idx_map[VAConfigAttribEncMaxRefFrames] ].value != VA_ATTRIB_NOT_SUPPORTED)
```

Reply:
Hi Dvrogozh, I will modify the status from VA_STATUS_ERROR_UNSUPPORTED_ENTRYPOINT and VA_STATUS_ERROR_UNSUPPORTED_PROFILE to  VA_ATTRIB_NOT_SUPPORTED in every areas.

Reply:
This is not about status at all...

Reply:
Done.